Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+node('D4R') {
+  currentBuild.result = 'SUCCESS'
+
+  def batFolder = "..\\DynamoRevit\\src\\scripts\\ci"
+
+  def branch = env.BRANCH_NAME
+
+  try {
+
+    stage('Checkout') {
+      echo "Checkout ..."
+      try {
+        checkout([$class: 'GitSCM', branches: [[name: '*/RC1.3.3_master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: '../Dynamo'], [$class: 'CleanBeforeCheckout'], [$class: 'GitLFSPull']], gitTool: 'default', submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'aecbuild', url: 'https://github.com/DynamoDS/Dynamo.git']]])
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: '../DynamoRevit'] , [$class: 'CleanBeforeCheckout'], [$class: 'GitLFSPull']], gitTool: 'default', submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'aecbuild', url: 'https://git.autodesk.com/Dynamo/DynamoRevit.git']]])
+      } catch(err) {
+        echo "Git Checkout Failed: $err"
+        currentBuild.result = 'FAILURE'
+        throw err
+      }
+    }    
+
+    stage('Revit API Download') {
+      echo "RevitAPI and RevitAPIUI download and extract ..."
+      try {
+        bat batFolder + '\\' + 'RevitAPInuGet.bat'
+      } catch(err) {
+        echo "RevitAPI or RevitAPIUI Download or Extract Error: $err"
+        currentBuild.result = 'FAILURE'
+        throw err
+      }
+    }    
+
+    stage('Build') {
+      echo "Build..."
+      try {
+        bat batFolder + '\\' + 'BuildSolutions.bat'
+      } catch(err) {
+        echo "Dynamo or DynamoRevit Build Failed: $err"
+        currentBuild.result = 'FAILURE'
+        throw err
+      }
+    }
+
+    if (branch == 'master' || branch.indexOf('release') == 0) {
+      stage('Deployment to Artifactory') {
```

Comment:
please remove the "|| branch.indexOf('release') == 0" as we only support the master branch right now.